src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ResourceHeatMapGraphTest.java

```diff
@@ -506,17 +506,13 @@ private void verifyFullNodeTemperatureProfile(String resp) {
           break;
         case Heap_AllocRate:
           break;
-        case IO_WriteSyscallRate:
-          break;
-        case IO_READ_SYSCALL_RATE:
-          break;
       }
     }
   }
 
   private void verifyCpuDimension(JsonObject cpuObject) {
```

Comment:
Can we add a similar method to test shard_bytes for a cluster level RCA and the node level one ?

Reply:
Done